api/GovStack_Registration_BB_API-registration-0.0.1-resolved.json

```
@@ -0,0 +1,163 @@
+{
+	"openapi": "3.0.0",
+	"info": {
+		"title": "Registration BB API descriptions",
+		"description": "API from Registration BB",
+		"termsOfService": "#",
+		"contact": {
+			"email": "ingmar.vali@gmail.com"
+		},
+		"license": {
+			"name": "Apache 2.0",
+			"url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+		},
+		"version": "0.0.1"
+	},
+	"servers": [{
+		"url": "http://ss2.egovstack.net/r1/eGovStack/COM/11222456/RegistrationBB/creg/",
+		"description": "Information Mediator BB Security server"
+	}],
+
+	"paths": {
+
+
+		"/api/process-definition/{key}/start-with-form": {
+			"post": {
+				"tags": ["application-registration"],
+				"summary": "Start application process instance with form",
+				"description": "Creates application process instance",
```

```suggestion
				"description": "Creates application process instance for an individual client's request who can do the registration process through the created form instance.",
```
Is that its purpose? (I am still trying to wrap my head around it overall)

---

I think it is like this: 
Process- Service that is created to capture users information. 
Instance- Application file created in the service.